app/src/main/java/io/reark/rxgithubapp/data/stores/NetworkRequestStatusStore.java

```diff
@@ -68,6 +72,11 @@ public void put(@NonNull NetworkRequestStatus item) {
         super.put(item);
     }
 
+    @Override
+    protected Observable<ArrayList<ContentProviderOperation>> groupOperations(Observable<ContentProviderOperation> source) {
+        return source.map(operation -> new ArrayList<ContentProviderOperation>(){{ add(operation); }});
```

Comment:
If the entire thing would return `Observable<List<>>` then this thing could return `source.map(Collections::singletonList);`